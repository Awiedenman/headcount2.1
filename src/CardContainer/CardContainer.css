.card-container {
  display: flex;
  flex-wrap: wrap;
  background-image: url('../images/memphis-colorful.zip')
}