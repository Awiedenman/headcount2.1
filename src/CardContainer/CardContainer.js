import React from 'react';
// import { Card } from '../Card/Card';

export const CardContainer = ( props ) => {
  return (
    <div>
      CardContainer
    </div>
  );
};