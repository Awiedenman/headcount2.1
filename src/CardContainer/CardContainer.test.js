import React from 'react';
import { CardContainer } from './CardContainer';
import { shallow } from 'enzyme';

describe('CardContainer', () => {
  it('matches a snapshot of the rendered card container', () => {
    const mockData = [{
      location: 'Colorado', stats : {
        2004: 0.24, 2005: 0.278, 2006: 0.337, 2007: 0.395}
    }];
    const wrapper = shallow( 
      <CardContainer 
        allDistricts = { mockData }
      />
    );
    
    expect(wrapper).toMatchSnapshot();
  });
});