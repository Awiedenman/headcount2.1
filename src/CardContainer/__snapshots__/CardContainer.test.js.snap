// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardContainer matches a snapshot of the rendered card container 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    allDistricts={
        Array [
            Object {
              "location": "Colorado",
              "stats": Object {
                "2004": 0.24,
                "2005": 0.278,
                "2006": 0.337,
                "2007": 0.395,
              },
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Unknown
          location="Colorado"
          stats={
                    Object {
                              "2004": 0.24,
                              "2005": 0.278,
                              "2006": 0.337,
                              "2007": 0.395,
                            }
          }
/>,
      ],
      "className": "card-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "0",
        "nodeType": "function",
        "props": Object {
          "location": "Colorado",
          "stats": Object {
            "2004": 0.24,
            "2005": 0.278,
            "2006": 0.337,
            "2007": 0.395,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Unknown
            location="Colorado"
            stats={
                        Object {
                                    "2004": 0.24,
                                    "2005": 0.278,
                                    "2006": 0.337,
                                    "2007": 0.395,
                                  }
            }
/>,
        ],
        "className": "card-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "0",
          "nodeType": "function",
          "props": Object {
            "location": "Colorado",
            "stats": Object {
              "2004": 0.24,
              "2005": 0.278,
              "2006": 0.337,
              "2007": 0.395,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
